rcl_interfaces/msg/Log.msg

```diff
@@ -1,5 +1,14 @@
 ##
 ## Severity level constants
+## 
+## These logging levels follow the Python Standard
+## https://docs.python.org/3/library/logging.html#logging-levels
+## And are implemented in rcutils as well
+## https://github.com/ros2/rcutils/blob/35f29850064e0c33a4063cbc947ebbfeada11dba/include/rcutils/logging.h#L164-L172
+## This leaves space for other standard logging levels to be inserted in the middle in the future,
+## as well as custom user defined levels.
+## Since there are several other logging enumeration standard for different implementations,
+## other logging implementations may need to provide level mappings to match their internal implementations.
```

Comment:
Out of curiosity: why two `#`? Are these comments still being extracted properly and used in the generated code without any leading `#`?

Reply:
I was just copying the existing style.